ecology.yaml

```diff
@@ -0,0 +1,525 @@
+---
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: transformation
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: anova
+    owner: lecorguille
+    tool_panel_section_label: 'Statistics'
+
+  - name: heatmap
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: univariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: multivariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: 'Annotation'
+
+  - name: add_input_name_as_column
+    owner: mvdbeek
+    tool_panel_section_label: 'Text Manipulation'
+ 
+  - name: regex_replace
+    owner: kellrott
+    tool_panel_section_label: 'Text Manipulation'
+
+  - name: find_in_reference
+    owner: jjohnson
+    tool_panel_section_label: 'Filter and Sort'
+
+  - name: normalize_dataset
+    owner: bebatut
+    tool_panel_section_label: 'Statistics'
+    
+  - name: frogs_2_0_0
+    owner: frogs
+    tool_panel_section_label: 'Find Rapidly OTUs from NGS'
+    
+  - name: frogs_3_1_0
+    owner: frogs
+    tool_panel_section_label: 'Find Rapidly OTUs from NGS'
+
+
+  - name: csv_to_tabular
```

Comment:
this is now a build-in function in Galaxy afaik

Reply:
True! I didn't know the last version embedded this! Too cool! Groovy!!!!

<img width="745" alt="csvTotab" src="https://user-images.githubusercontent.com/7910679/58749029-eb4ec880-8480-11e9-8ab7-ccfb283e62d5.PNG">

Reply:
Old tools removed!